Samples~/AvatarCreatorSamples/AvatarCreatorWizard/Scripts/UI/Buttons/AssetButton.cs

```diff
@@ -25,8 +27,14 @@ public void SetColor(string colorHex)
 
         public void SetIcon(Texture texture)
         {
+            if (rawImageRectTransform == null)
+            {
+                rawImageRectTransform = icon.GetComponent<RectTransform>();
```

Comment:
What about converting variable to property and defining this inside of the property: